NAMESPACE

```diff
@@ -4,7 +4,6 @@ S3method(head,magpie)
 S3method(print,magpie)
 S3method(str,magpie)
 S3method(tail,magpie)
-S3method(tibble::as_tibble,magpie)
```

Comment:
This seems strange to me, do you know why this was changed (assuming it was automatically changed by buildLibrary/devtools::document)?

Reply:
are your R packages up-to-date and are you using a recent R version?

Reply:
My R version is 4.3.2 and my packages are almost up-to-date (as of last week or the week before).

Reply:
When I run `roxygen2::roxygenize()` on your fork I get the line you deleted here back, so I'd rather leave it in :thinking:

Reply:
I'm also on R 4.3.2 and just updated my packages. Also I don't really see why your manual changes would imply this change

Reply:
do you have all suggested packages installed, especially tibble?